boards/x86/arduino_101/arduino_101.dts

```diff
@@ -66,6 +66,7 @@
 		#address-cells = <1>;
 		#size-cells = <1>;
 		compatible = "winbond,w25q16", "jedec,spi-nor";
+		label = "jedic,spi-nor#0";
```

Comment:
Any reason not to have this one just be W25Q16?

Reply:
No.